source/Tests/Testing/Integration/IntegrationTestContainerAnalyzerFixture.cs

```diff
@@ -204,28 +204,107 @@ public class TestClass : IntegrationTest
         public async Task DetectsIntegrationTestContainerClassesWithMembersThatAreNotTypesOrPrivateMethods()
         {
             var container = @"
+using System.Collections.Generic;
```

Comment:
I think splitting these into separate tests would be better. That will allow the next person to more easily understand how the changed the analyser.

Reply:
Good call, will do